### PR TITLE
Major documentation updates (k3s, kas)

### PR DESCRIPTION
**Pull request by mikehaller**
This PR updates the documentation regarding build tool (kas) and SDV.EDGE container runtime migration from Kubernetes to k3s.

It also includes general updates to feature list, roadmap etc.

### CODE REVIEW

File: content/en/docs/app-deployment/velocitas/_index.md [hunk 1]
```diff
@@ -5,37 +5,100 @@ date: 2022-05-09T14:24:56+05:30
 
 For local deployment without a cloud backend, deployment specifications for [Eclipse Velocitas](https://projects.eclipse.org/projects/automotive.velocitas) Vehicle Apps may be put directly onto the device.
 
-- Create a Kubernetes deployment specification for your vehicle app
-- Copy the specification file to the device, e.g. `scp -P 2222 myapp.yaml root@localhost`
-- Apply the specification: `ssh -p 2222 root@localhost /usr/local/bin/kubectl apply -f myapp.yaml`
+- Create a Kanto deployment specification for your vehicle app. You may copy one of the existing JSON files in `/data/containers/manifests` as a template
+- Copy the specification file to the device, e.g. `scp -P 2222 myapp.json root@localhost:/data/containers/manifests/`
```
Comment (stlachev) [inline content/en/docs/app-deployment/velocitas/_index.md:9]: /data/containers/manifests -> /data/var/containers/manifests
on both places

File: content/en/docs/app-deployment/velocitas/_index.md [hunk 2]
```diff
@@ -5,37 +5,100 @@ date: 2022-05-09T14:24:56+05:30
...
+        "restart_policy": {
+            "maximum_retry_count": 0,
+            "retry_timeout": 0,
+            "type": "unless-stopped"
+        },
+        "runtime": "io.containerd.runc.v2",
+        "extra_hosts": [],
+        "port_mappings": [
+            {
+              "protocol": "tcp",
```
Comment (stlachev) [inline content/en/docs/app-deployment/velocitas/_index.md:50]: container_port must defer from rest something like
"container_port": 30151,

File: content/en/docs/build/concepts.md [hunk 3]
```diff
@@ -4,41 +4,60 @@ date: 2022-05-09T14:24:56+05:30
...
 
-# Recipes for Kubernetes Pods
+## Recipes for containerized components
 
-The SDV stack is based on a Kubernetes Control Plane and the intention is to have as much components containerized as possible, to ensure a high degree of isolation and updateability.
+The SDV.EDGE stack is based on a containerized architecture and the intention is to have as much components containerized as possible, to ensure a high degree of isolation and updateability. To ensure some degree of flexibility and control, certain core components may also be installed as native services.
 
-To automatically deploy the pods of the SDV reference implementation and example applications and services, the build configurations will deploy a couple of Pod specifiction files into the [auto-deployment folder of k3s](https://rancher.com/docs/k3s/latest/en/advanced/#auto-deploying-manifests).
+To automatically deploy the containers of the SDV reference implementation and example applications and services, the build configurations will deploy a couple of deployment specifiction files into the auto-deployment folder `/data/containers/manifests`.
```
Comment (stlachev) [inline content/en/docs/build/concepts.md:42]: /data/var/containers/manifests

File: content/en/docs/device-provisioning/manual-provisioning.md [hunk 4]
```diff
@@ -6,81 +6,46 @@ weight: 2
...
-  Type              Status
-  Initialized       True 
-  Ready             True 
-  ContainersReady   True 
-  PodScheduled      True 
-```
+- Generate the device certificate (eg using openssl) and sign it with your CA.
+- Log in to Azure Portal, Go to Azure Iot Hub and create a new device
+- Select the proper authentication type, e.g. X.509 Self-signed or X.509 CA Signed
+- Copy the device certificate (cert file and key file) to the device to `/data/certificate`
```
Comment (stlachev) [inline content/en/docs/device-provisioning/manual-provisioning.md:12]: /data/var/certificates

File: content/en/docs/device-provisioning/manual-provisioning.md [hunk 5]
```diff
@@ -6,81 +6,46 @@ weight: 2
...
+
+For the proper device authentication, the device management backend authority needs to issue a device-specific certificate and sign it. This is a complex process and subject to the specific situation.
+
+For the Leda quickstart images, the software configuration is prepared with dummy certificates which need to be replaced.
+
+> ATTENTION: The Leda example device certificates are public and insecure, they only serve demonstration purposes. You need to replace the intermediate certificates and device certificates with your own.
+
+- Generate a device certificate using openssl
+- Sign it with your intermediate CA certificate
+- Put it into `/data/certificate/`
```
Comment (stlachev) [inline content/en/docs/device-provisioning/manual-provisioning.md:41]: /data/var/certificates

File: content/en/docs/leda-incubator/self-update-agent/_index.md [hunk 6]
```diff
@@ -24,6 +24,32 @@ SUA is using the RAUC framework via D-Bus calls, but it is designed in a way tha
...
+
+### Container Installation
+
+When the component can be deployed on the more generic compute module, where a container runtime is available, the self update agent can also be deployed as a container. This is the default for the Eclipse Leda quickstart images.
+
+Initial deployment and configuration steps are:
+
+- On first start, the auto deployment will deploy and start the self update agent container automatically.
+- The self update agent will then connect to the locally running mosquitto server, awaiting update requests.
+- The default configuration will use the location `/data/self-update-images/` to write and read update files.
```
Comment (stlachev) [inline content/en/docs/leda-incubator/self-update-agent/_index.md:51]: /data/selfupdates

File: content/en/docs/app-deployment/velocitas/_index.md (hunk 2 above)
Comment (mikehaller) [reply content/en/docs/app-deployment/velocitas/_index.md:50]: Okay, fixed

File: content/en/docs/app-deployment/velocitas/_index.md (hunk 1 above)
Comment (mikehaller) [reply content/en/docs/app-deployment/velocitas/_index.md:9]: Fixed

File: content/en/docs/app-deployment/velocitas/_index.md [hunk 7]
```diff
@@ -5,37 +5,100 @@ date: 2022-05-09T14:24:56+05:30
...
-        - containerPort: 50008
-      env:
-        - name: SEAT_SERVICE_ADDRESS
-          value: "edgehost:50051"
-  imagePullSecrets:
-  - name: my-private-container-registry-secret
+To update an existing container when the configuration has changed, delete the container and restart kanto-auto-deployer:
+
+    kanto-cm remove -n myapp-example
+    # Edit /data/containers/manifests/myapp.json
```
[the remark this answers is not in the document]
Comment (mikehaller) [reply content/en/docs/app-deployment/velocitas/_index.md:15]: Fixed

File: content/en/docs/leda-incubator/self-update-agent/_index.md (hunk 6 above)
Comment (mikehaller) [reply content/en/docs/leda-incubator/self-update-agent/_index.md:51]: Fixed

File: content/en/docs/device-provisioning/manual-provisioning.md (hunk 5 above)
Comment (mikehaller) [reply content/en/docs/device-provisioning/manual-provisioning.md]: Fixed

**Pull request merged**